public_dropin_notebook_environments/python39_notebook/env_info.json

```diff
@@ -1,9 +1,9 @@
 {
-  "id": "6386dc1159c606b0d8beddc7",
-  "name": "[DataRobot] Python 3.9 Notebook Drop-In",
-  "description": "This template environment can be used to create Python 3.9 notebook environments.",
-  "programmingLanguage": "python",
-  "environmentVersionId": "6386dc2159c606b0d8beddc8",
-  "isPublic": true,
-  "useCases": ["notebook"]
+	"id": "6386dc1159c606b0d8beddc7",
+	"name": "[DataRobot] Python 3.9 Notebook Drop-In",
+	"description": "This template environment can be used to create Python 3.9 notebook environments.",
+	"programmingLanguage": "python",
+	"environmentVersionId": "6386dc2159c606b0d8beddc4",
```

Comment:
Looks like you just manually bump or change the last character.
Probably generating a new one is safer. 
Just make sure there is no  clash